urlman.py

```diff
@@ -91,18 +116,15 @@ def get_value(self, key, args, kwargs):
         except ValueError:
             pass
         else:
-            # If it's a callable, call it with the outer instance as self
```

Comment:
I couldn't for the life of me find a case in which this if block is true, so I removed it without tests failing. In case you can provide a case, please let me know.